src/site/content/en/blog/origin-private-file-system/index.md

```diff
@@ -0,0 +1,406 @@
+---
+title: 'The origin private file system'
+subhead: >
+  The File System Standard introduces an origin private file system (OPFS) as a storage endpoint private to the origin of the page and not visible to the user that provides optional access to a special kind of file that is highly optimized for performance.
+date: 2023-03-27
```

Comment:
let's give this an updated publication date as we're celebrating the time it became interoperable which would be weird otherwise. I'd suggest putting a line at the bottom (This article was originally published at...)

Reply:
Done.